--- /usr/local/lib/node_modules/oref0/bin/oref0-online.sh       2017-12-02 10:44:58.702380978 -0800
+++ /usr/local/lib/node_modules/oref0/bin/oref0-online.sh       2018-05-24 15:33:51.097106724 -0700
@@ -114,7 +114,7 @@
     # loop over as many MACs as are provided as arguments
     for MAC; do
         #echo -n "At $(date) my public IP is: "
-        if ! check_ip >/dev/null; then
+        if ! check_ip >/dev/null && ! has_ip bnep0; then
             echo; echo "No Internet access detected, attempting to connect BT to $MAC"
             oref0-bluetoothup
             sudo bt-pan client $MAC -d
